core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -138,6 +139,36 @@ public void testNullStatusOnKraftCommandAlterClientMetrics() {
         assertEquals("Completed updating config for client-metric cm.", message);
     }
 
+    @ClusterTest
+    public void testAddConfigKeyValuesUsingCommand() throws Exception {
+
+        try (Admin client = cluster.admin()) {
+            NewTopic newTopic = new NewTopic("test-topic", 1, (short) 1);
+            client.createTopics(Collections.singleton(newTopic)).all().get();
+        }
+
```

Comment:
We need to make sure topic is created here.
You can use https://github.com/apache/kafka/blob/53d8316b5da7d8355331aa2d50193748c8261706/test-common/test-common-api/src/main/java/org/apache/kafka/common/test/api/ClusterInstance.java#L250

Reply:
OK.Thanks.I will fix it.